src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -8,6 +8,7 @@
 #include "providers/twitch/PartialTwitchUser.hpp"
 #include "providers/twitch/TwitchChannel.hpp"
 #include "singletons/Resources.hpp"
+#include "singletons/Settings.hpp"

```

Comment:
Don't have to include this anymore since you're not using settings

Reply:
removed